extensions/admin-tool-ext-win/src/test/utils.test.ts

```diff
@@ -77,6 +80,10 @@ const tableSchema = 'tbl\'sch';
 const escapedTableSchema = doubleEscapeSingleQuotes(tableSchema);
 
 describe('buildUrn Method Tests', () => {
+	it('Azure Server Name is trimmed correctly', async function(): Promise<void> {
+		should(await buildUrn(fullAzureServerName, undefined)).equal(`Server[@Name=\'${escapedAzureServerName}\']`);
```

Comment:
Don't Urns work without a Name attribute on the Server? I thought SFC would grab the server name out of the connection if it really needs it.

Reply:
Ah jeez, good catch. That simplifies this so much!